src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -160,11 +171,23 @@ const SidebarView = View.extend({
     this.getRegion('info').empty();
   },
   onSave() {
-    this.getRegion('save').empty();
+    this.render();
   },
   onCancel() {
     this.triggerMethod('close', this);
   },
+  onConfirmDelete() {
+    const modal = Radio.request('modal', 'show:small', {
+      bodyText: intl.admin.sidebar.clinician.clinicianSidebarViews.sidebarView.deleteModal.bodyText,
```

Comment:
could probably make a `const i18n = intl.admin.sidebar.clinician.clinicianSidebarViews;` at the top